contrib/Dockerfile

```diff
@@ -29,8 +29,6 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 ADD linux /usr/local/bin
 
-EXPOSE 8080
```

Comment:
I think it should be a separate PR.

Reply:
Good idea. https://github.com/dgraph-io/dgraph/pull/8733